doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should respond within 3 seconds.
+6. Should be usable by anyone who is able to type into system in English.
+7. Should be usable by a novice who has little technology knowledge.
```

Comment:
NFR must be specific, thus you can mention something like, `novice user can use all features of the app within the first 10 minutes.`